src/ngx_http_lua_socket_tcp.c

```diff
@@ -4891,6 +4967,7 @@ ngx_http_lua_socket_tcp_upstream_destroy(lua_State *L)
         return 0;
     }
 
+    luaL_unref(L, LUA_REGISTRYINDEX, u->bind_tbl_ref);
```

Comment:
there are four connection closing handler need to call `luaL_unref ` after i review the code. 

`ngx_http_lua_socket_shutdown_pool`, `ngx_http_lua_cleanup_conn_pools`, `ngx_http_lua_socket_keepalive_close_handler`, `ngx_http_lua_socket_tcp_finalize` .

please advise，i can't sure if my understanding is correct.

![snip20160819_2](https://cloud.githubusercontent.com/assets/6814606/17814639/57538d34-6663-11e6-9a53-f433e2481346.jpg)